src/Pages/DistributedUptimeStatus/Status/Components/TimeframeHeader/index.jsx

```
@@ -0,0 +1,39 @@
+import { Stack, Button, ButtonGroup } from "@mui/material";
+import { RowContainer } from "../../../../../Components/StandardContainer";
+import { useTheme } from "@emotion/react";
+
+const TimeFrameHeader = ({ timeFrame, setTimeFrame }) => {
```

ESLint

[error] 'timeFrame' is missing in props validation

(react/prop-types)